.github/workflows/dco-check.yml

```diff
@@ -0,0 +1,45 @@
+# Unprivileged first part of dco-report.yml.  Architecture and code from
+# https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+#
+# Invoke with:
+#
+#     name: DCO
+#     on:
+#       pull_request:
+#         branches: [main]
+#     permissions:
+#       contents: read
+#     jobs:
+#       check:
+#         name: Organization
+#         uses: openslide/.github/.github/workflows/dco-check.yml@main
+
+name: DCO (reusable)
+
+on:
+  workflow_call:
+
+jobs:
+  check:
+    name: Check signoffs
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
```

Comment:
there is an `actions/checkout@v3` but it is unclear to me what the differences really are. doesn't look like they are using semver

Reply:
Yeah, I used v2 by force of habit.  Fixed.